File: Motocitizen/src/motocitizen/fragments/BusinessCardActivityFragment.java
package motocitizen.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import motocitizen.main.R;

public class BusinessCardActivityFragment extends Fragment {

    public BusinessCardActivityFragment() {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View viewMain = inflater.inflate(R.layout.fragment_business_card, container, false);
        ImageView imageViewQrForum = (ImageView) viewMain.findViewById(R.id.imageViewQrForum);
        imageViewQrForum.setImageResource(R.drawable.qr_forum);
        return viewMain;
    }
}

File: Motocitizen/src/motocitizen/ui/fragments/BusinessCardActivityFragment.kt
package motocitizen.ui.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView

import motocitizen.main.R

class BusinessCardActivityFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val viewMain = inflater.inflate(R.layout.fragment_business_card, container, false)
        val imageViewQrForum = viewMain.findViewById(R.id.imageViewQrForum) as ImageView
        imageViewQrForum.setImageResource(R.drawable.qr_forum)
        return viewMain
    }
}
